public/scripts/extensions/shared.js

```diff
@@ -424,16 +431,26 @@ export class ConnectionManagerRequestService {
         }
 
         dropdown.empty();
-        dropdown.append('<option value="">Select a Connection Profile</option>');
+
+        // Create default option using document.createElement
+        const defaultOption = document.createElement('option');
+        defaultOption.value = '';
+        defaultOption.textContent = 'Select a Connection Profile';
+        defaultOption.setAttribute('data-i18n', 'Select a Connection Profile');
```

Comment:
That's okay. But when not using jQuery (thank god), there is a neat and easy thing to set data attributes.
```suggestion
        defaultOption.dataset.i18n = 'Select a Connection Profile';
```

Reply:
Fixed